<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlvSivu</class>
 <widget class="QWidget" name="AlvSivu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Vero&amp;kausi</string>
            </property>
            <property name="buddy">
             <cstring>kausiCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="kausiCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Alv-kirjaukset lukittu</string>
            </property>
            <property name="buddy">
             <cstring>viimeisinEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="viimeisinEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Maksuperusteinen alv</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
            <item>
             <widget class="QLabel" name="maksuAlv">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="maksuperusteNappi">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../pic/pic.qrc">
                <normaloff>:/pic/ratas.png</normaloff>:/pic/ratas.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Alv-ilmoitus alkaa</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="erapaivaLabel">
              <property name="text">
               <string>xxx</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Eräpäivä</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDateEdit" name="alkaaEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Alv-ilmoitus päättyy</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDateEdit" name="paattyyEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="tilitaNappi">
            <property name="text">
             <string>&amp;Tee alv-tilitys</string>
            </property>
            <property name="icon">
             <iconset resource="../pic/pic.qrc">
              <normaloff>:/pic/verotilitys.png</normaloff>:/pic/verotilitys.png</iconset>
            </property>
            <property name="shortcut">
             <string>F11</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="ilmoituksetView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="tilitysNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Näytä tilitys</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/print.png</normaloff>:/pic/print.png</iconset>
       </property>
       <property name="shortcut">
        <string>F9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaTilitysNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Poista tilitys</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/roskis.png</normaloff>:/pic/roskis.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
